src/lib/openapi/spec/project-link-templates-schema.ts

```diff
@@ -0,0 +1,29 @@
+import type { FromSchema } from 'json-schema-to-ts';
+
+export const projectLinkTemplateSchema = {
+    $id: '#/components/schemas/projectLinkTemplateSchema',
+    type: 'object',
+    description:
+        'A template for a link that can be automatically added to new feature flags.',
+    required: ['template'],
+    properties: {
+        title: {
+            type: 'string',
+            description: 'The title of the link.',
+            example: 'Code search',
+            nullable: true,
+        },
+        template: {
+            type: 'string',
+            description:
+                'The URL to use as a template. Can contain {{project}} or {{flag}} as placeholders.',
```

Comment:
flag or feature? I think feature is more common at least in our code

Reply:
yes, because we had 'feature' as in our models in the beginning, and UI is moving in this direction. should backend keep old convention?